src/generator.js

```diff
@@ -0,0 +1,14 @@
+const HandleBars = require("handlebars");
+var path = require('path');
+var appDir = path.dirname(require.main.filename);
+const { readFile } = require('./file');
+
+function template(payload) {
+    const file = readFile(appDir + '/src/templates/br.html')
```

Comment:
Maybe this `br` can be passed to payload as `locale` prop comming right from the args as `-l` or `--locale` and have the default setted to `br`.